pkg/timeutil/timeutil.go

```diff
@@ -20,9 +24,32 @@ func Date(datetime time.Time) time.Time {
 func LastWeekday(weekday time.Weekday) time.Time {
 	date := Date(time.Now())
 
-	for date.Weekday() != weekday {
+	for date.Weekday() !=  weekday{
 		date = date.AddDate(0, 0, -1)
 	}
 
 	return date
 }
+
+// StringToWeekday transforms a string to time.Weekday
+// it returns Monday if wrong day specified in config.toml
+func StringToWeekday(weekday string) (time.Weekday, error){
+	switch strings.ToLower(weekday) {
+		case "monday":
+			return time.Monday, nil
+		case "tuesday":
+			return time.Tuesday, nil
+		case "wednesday":
+			return time.Wednesday, nil
+		case "thursday":
+			return time.Thursday, nil
+		case "friday":
+			return time.Friday, nil
+		case "saturday":
+			return time.Saturday, nil
+		case "sunday":
+			return time.Sunday, nil
+	}
+
+	return time.Sunday, fmt.Errorf("Invalid first week day '%s' specifid in config.toml. Sunday is used as default", weekday)
```

Comment:
I'd go for this exact message format instead, including the package name, which also passes golint checks for the error not starting with a capital letter.

> timeutil: Invalid weekday '%s' given

No more, no less. This function is not exclusive to use in configuration. Also, Sunday is not a default, it's a value that just happens to be a valid `time.Weekday` (i.e. we have to return an actual `time.Weekday` instance because that's what the return type specifies, but we don't care what it is, it's not a default, it's inconsequential).